OutOfSchool/OutOfSchool.WebApi/Common/GettingUserProperties.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Security.Authentication;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Http;
+using OutOfSchool.Common;
+using OutOfSchool.Common.Extensions;
+using OutOfSchool.Services.Enums;
+
+namespace OutOfSchool.WebApi.Common
+{
+    public static class GettingUserProperties
+    {
+        public static string GetUserId(HttpContext httpContext)
+        {
+            var userId = GetUserId(httpContext?.User)
+                ?? throw new AuthenticationException($"Can not get user's claim {nameof(IdentityResourceClaimsTypes.Sub)} from HttpContext.");
+
+            return userId;
+        }
+
+        public static string GetUserId(ClaimsPrincipal user)
+        {
+            return user.GetUserPropertyByClaimType(IdentityResourceClaimsTypes.Sub);
```

Comment:
Possible place of `NullReferenceException`. Please at least use elvis-operator - `?.` (seems to be the choice conforming call from line 15) or any other countermeasure.

Reply:
Done